src/core/metrics/sources/nginx_plus.go

```diff
@@ -827,10 +877,48 @@ func createStreamPeerMap(peers []plusclient.StreamPeer) map[string]plusclient.St
 	return m
 }
 
-func boolToFloat64(mybool bool) float64 {
-	if mybool {
+func boolToFloat64(myBool bool) float64 {
+	if myBool {
 		return valueFloat64One
 	} else {
 		return valueFloat64Zero
 	}
 }
+
+func (c *NginxPlus) getLatestAPIVersion(ctx context.Context, httpClient *http.Client, endpoint string) (int, error) {
```

Comment:
Do you need to pass httpClient as an argument? Could you create it at the start of the function instead?